artix7/primitives/dsp48e1/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_subdirectory(nreg)
```

Comment:
Is there a way to auto-generate the subdirectory stuff?

Reply:
Agreed, explicit over implicit.  Globing in CMake has unintended consequences, so lets avoid it if possible.